internal/cmd/deploy/deploy.go

```diff
@@ -175,6 +173,23 @@ YAML:
 			}
 		}
 	},
+	PostRun: func(cmd *cobra.Command, args []string) {
+		params = nil
+	},
+}
+
+func changeSetHasNoChanges(msg string) bool {
+	// mesages returned as error when the change set is empty
+	noChangeFoundMsg := []string{
+		"The submitted information didn't contain changes. Submit different information to create a change set.",
+		"No updates are to be performed.",
```

Comment:
In the mock it's lower case, which won't match, is that intentional?

Reply:
yea it is intentional,explained it in the previous comment

Reply:
Might be better to suppress the linter check instead of using the formatting strings to fool it. https://staticcheck.io/docs/configuration/#line-based-linter-directives

Reply:
i did not know that, do you want me to add that ?

Reply:
Yes, if you have time, I think it would make the code more readable. Appreciate the contribution!

Reply:
done! It looks a lot cleaner now, thanks for the pointer. Love the cli and fixing these little bugs is the least I can do : )